lib/CL/pocl_llvm_build.cc

```diff
@@ -1045,8 +1045,13 @@ int pocl_invoke_clang(cl_device_id Device, const char** Args) {
 
   const char **ArgsEnd = Args;
   while (*ArgsEnd++ != nullptr) {}
+  llvm::SmallVector<const char*, 0> ArgsArray(Args, ArgsEnd);
 
-  llvm::ArrayRef<const char*> ArgsArray(Args, ArgsEnd);
+  std::string ld_path;
+  if (const char* ld = pocl_get_path("LD", nullptr)) {
```

Comment:
```suggestion
  if (const char* LDOverride = pocl_get_path("LD", nullptr)) {
```
(or similar).

Otherwise, LGTM!